website/pages/docs/k8s/installation/multi-cluster/kubernetes.mdx

```diff
@@ -0,0 +1,388 @@
+---
+layout: docs
+page_title: Federation Between Kubernetes Clusters
+sidebar_title: Federation Between Kubernetes Clusters
+description: >-
+  Federating multiple Kubernetes clusters.
+---
+
+# Federation Between Kubernetes Clusters
+
+-> **1.8.0+:** This feature is available in Consul versions 1.8.0 and higher
+
+~> This topic requires familiarity with [Mesh Gateways](/docs/connect/mesh_gateway) and [WAN Federation Via Mesh Gateways](/docs/connect/wan-federation-via-mesh-gateways).
+
+-> Looking for a step-by-step guide? Please follow our Learn Guide: [Secure and Route Service Mesh Communication Across Kubernetes](https://learn.hashicorp.com/consul/kubernetes/mesh-gateways).
+
+This page describes how to federate multiple Kubernetes clusters.
```

Comment:
Should we add a use case sentence for federation here, maybe something like "A common scenario for needing federation is to be able to run the same application across multiple Kubernetes clusters that span datacenters or regions" ?

Reply:
I've listed use cases in the overview page for this multi-cluster section. Do you think I should add it here as well?

```
# Multi-Cluster Federation Overview

In Consul, federation is the act of joining two or more Consul datacenters.
When datacenters are joined, Consul servers in each datacenter can communicate
with one another. This enables the following features:

- Services on all clusters can make calls to each other through Consul Service Mesh.
- [Intentions](/docs/connect/intentions) can be used to enforce cross-cluster rules about which services can communicate.
- [L7 Routing Rules](/docs/connect/l7-traffic-management) can enable multi-cluster failover
  and traffic splitting.
- The Consul UI has a drop-down menu that lets you navigate between datacenters.

```

Reply:
I added a link to the overview:
```
This page describes how to federate multiple Kubernetes clusters. See [Multi-Cluster Overview](/docs/k8s/installation/multi-cluster/overview)
for more information on use-cases and how it works.
```